spec/index.html

```diff
@@ -310,7 +310,9 @@ <h2>Terminology</h2>
       with <code>_:</code> that is used as an identifier for an
       <a>blank node</a>. <a>Blank node identifiers</a>
       are typically implementation-specific local identifiers; this document
-      specifies an algorithm for deterministically specifying them.</dd>
+      specifies an algorithm for deterministically specifying them.
+      A blank node not given an explicit identier is called an "unlabeled blank node."
```

Comment:
Blank nodes always being nodes without identifiers, this seems a bit redundant, but ... I guess it's necessary in this mishmosh.
```suggestion
      A blank node not given an explicit identifier is called an "unlabeled blank node."
```